scripts/backend_test_shards.json

```diff
@@ -205,7 +205,8 @@
         "core.domain.app_feedback_report_constants_test",
         "scripts.linters.warranted_angular_security_bypasses_test",
         "scripts.create_expression_parser_test",
-        "scripts.run_frontend_tests_test"
+        "scripts.run_frontend_tests_test",
+        "scripts.check_overall_backend_test_coverage_test"
```

Comment:
Shard 3 is now taking more than 30 mins to run ([log](https://github.com/oppia/oppia/runs/7718745427?check_suite_focus=true)). Can you add these to shard 2 instead, which is only taking 22 mins?

Reply:
Done!